### PR TITLE
[Radio ML] update layer configs

### PR DESCRIPTION
**Pull request by azizb-xlnx**
Update VGG10 RadioML to FINN v0.10.0 and change layers to RTL

### CODE REVIEW

File: build/vgg10-radioml/folding_config/ZCU104_folding_config.json [hunk 1]
```diff
@@ -1,136 +1,1193 @@
 {
   "Defaults": {},
-  "FMPadding_hls_0": {
-    "SIMD": 2
+  "StreamingFIFO_rtl_0": {
```
Comment (auphelia) [inline build/vgg10-radioml/folding_config/ZCU104_folding_config.json:3]: If you specify the FIFOs in the folding config json, you don't need to set `auto_fifo_depths=True,` in `build.py`. I would do one or the other.

File: build/vgg10-radioml/folding_config/ZCU104_folding_config.json [hunk 2]
```diff
@@ -1,136 +1,1193 @@
...
+    "inFIFODepths": [
+      0
+    ],
+    "outFIFODepths": [
+      0
+    ]
+  },
+  "FMPadding_rtl_0": {
+    "SIMD": 2,
+    "inFIFODepths": [
```
Comment (auphelia) [inline build/vgg10-radioml/folding_config/ZCU104_folding_config.json:16]: The `inFIFODepths` and `outFIFODepths` parameters are set by the automatic FIFO sizing and don't necessarily need to be part of the folding config json. Removing them, might make the folding configuration a little bit shorter and easier to read.

**Review by auphelia — CHANGES_REQUESTED** [review]

File: build/vgg10-radioml/folding_config/ZCU104_folding_config.json (hunk 1 above)
Comment (azizb-xlnx) [reply build/vgg10-radioml/folding_config/ZCU104_folding_config.json:3]: Removed auto sizing because this saves time.

**Review by auphelia — APPROVED** [review]
Looks good to me, thanks @azizb-xlnx

**Pull request merged**